src/utils/pdf.js

```diff
@@ -0,0 +1,22 @@
+const puppeteer = require('puppeteer');
+
+exports.createPdf = async function(html, options) {
```

Comment:
we are not using any transpiler hence lets just stick with 
```
module.exports = {
    async createPdf(...) { ... }
}
```

Reply:
done